README.md

```diff
@@ -24,7 +24,9 @@ The app itself is live on http://zhishi.andela.com/
 
 2. Then `npm install` all dependencies (and dev dependencies) required.
 
-3. Once the installation of dependencies is done, `npm start` should start up a server on port `8080`
+3. Edit the host file `sudo nano /etc/hosts` and add `127.0.0.1       zhishi-dev.andela.com` to the hosts
```

Comment:
Editing the hosts file is now only required when using Andela's auth, as a cookie is being set from Andela's domain! We may want to use a different approach, because as of right now, the same token is also being sent as a url query string, which absolves us of Andela's domain!! You may want to add this step as an optional, not required step!